webpack.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   entry: {
     app: ['./src/index.js']
   },
+  mode: 'development',
```

Comment:
Required in Webpack v4. Since we build this into out app via the web-apps Webpack config this can always just be development.